# Results {.unnumbered}

## Generated Networks {.unnumbered}

|     SC Layer 0 represents 0.63% (42/6,600) of the total nodes and 0.47% (166/34,892) of the total edges in the _S. cerevisiae_ dataset. AT Layer 0 represents 0.10% (28/27,462) of the total nodes and 0.14% (35/24,802) of the total edges in the _A. thaliana_ dataset. Layer 0 displays the largest discrepancy in average clustering values with AT layer 0 displaying an average clustering value of 0.35 while SC layer 0 displayed an average clustering value of 0.69. A total of 33 homologous edges were identified between the two layer 0's representing 19.87% (33/166) of the total _S. cerevisiae_ layer 0 edges, and 94.28% (33/35) of the total _A. thaliana_ layer 0 edges. 
|     SC Layer 1 represents 3.5% (236/6,600)  of the total nodes and 3.5% (1,250/34,892) of the total edges in the _S. cerevisiae_ dataset. AT Layer 1 represents 0.44% (121/27,462) of the total nodes and 1.64% (407/24,902) of the total edges in the _A. thaliana_ dataset. The average clustering coefficients for each organisms layer 1 were similar with _A. thaliana_ displaying a 0.59 clustering coefficient and _S. cerevisiae_ displaying a 0.64 clustering coefficient. A total of 101 homologous edges were identified between the two layer 1's representing 8.08% (101/1,250) of the total _S. cerevisiae_ layer 1 edges, and 24.81% (101/407) of the total _A. thaliana_ layer 1 edges. 

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r, echo=FALSE, fig.cap="S. cerevisiae layers. Layer 0 represents the initial node set. layers 1 and 2 represent the neighbors collected in the the first and second hop. All Layers represents both the initial node set and all neighbors collected from each hop",out.width="100%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/SC_layer_fig.png")
```


```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r,echo=FALSE,fig.cap="A. thaliana layers. Layer 0 represents the initial node set. layers 1 and 2 represent the neighbors collected in the the first and second hop. All Layers represents both the initial node set and all neighbors collected from each hop",out.width="100%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/AT_layer_fig.png")
```

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```


## Layer 0 Homologs {.unnumbered}
|     Analysis of the top 20 layer 0 _A. thaliana_ nodes by combined network importance score shows that the majority of the top scoring nodes in _A. thaliana_ layer 0 consist of proteins belonging to the HOPS complex, SNARE complex, and V-ATPase complex. All 6 HOPS proteins, 3 out of the 4 SNARE proteins, and 9 out of the 13 V-ATPase proteins were represented in the top 20 nodes of _A. thaliana_ layer 0. A comparison of each set of 20 proteins in the respective layer 0's shows that 13 out of the top 20 proteins in  _A. thaliana_ layer 0 are homologous to proteins in the top 20 proteins in _S. cerevisiae_ layer 0. Out of the 20 proteins in the _A. thaliana_ layer 0, two proteins were not members of the HOPS, SNARE, or V-ATPase complex. These two proteins (Vps34 and Vps15) are homologous to _S. cerevisiae_ Vps34 and Vps15 which have be shown to form complexes responsible for the conversion of membrane bound PI to PI3P.

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r,echo=FALSE, fig.cap="Layer 0 top 20 Network Importance. Each node in the graph represents a protein while each edge between two nodes represents a PPI between the nodes. The combined scores collumns represent the top 20 proteins in each layer 0 ranked by combined index score (see Network Importance Analysis in the Computational Methods Section). Each graph displays the entirety of each layer 0 with darker node color representing higher combined index score. Some edges between nodes may be obscured by overlapping neighboring nodes, however, if a node has any amount of coloring, it is connected by an edge to at least one other node",out.width="100%",out.height="70%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/L0_importance_updated_fig.png")
```

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r,echo=FALSE,fig.cap="Homologous edges between A. Thaliana and S. Cervisiae layer 0. Red nodes represent proteins in each layer with known homologs in the opposing layer. Red dashed edges connect each protein with its known homolog in the opposing layer",out.width="110%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/Multilayer_network.png",)
```

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

## Layer 1 Homologs {.unnumbered}
|     Analysis of the top 20 layer 1 _A. thaliana_ nodes shows a marked difference in comparison to layer 0. All  _A. thaliana_ layer 0 nodes in the top 20 are involved in lipid synthesis or modification (ie. phosphorylation or transferase activity). A single protein, Vps34, in shared between the top 20 nodes of each layer 1. Out of the top 20 _A. thaliana_ proteins, 10 have no identified homologs in yeast.

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r,echo=FALSE,fig.scrap="Layer 1 top 20 Network Importance",fig.cap="Each node in the graph represents a protein while each edge between two nodes represents a PPI between the nodes. The combined scores collumns represent the top 20 proteins in each layer 1 ranked by combined index score (see Network Importance Analysis in the Computational Methods Section). Each graph displays the entirety of each layer 1 with node color corresponding to the nodes combined index score (darker colors represent higher importance nodes).",out.width="100%",out.height="85%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/L1_importance_updated_fig.png")
```

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```
## Layer 2 Properties {.unnumbered}
SC Layer 2 represents 10.66% (704/6,600)  of the total nodes and 9.58% (3,345/34,892) of the total edges in the _S. cerevisiae_ dataset. AT Layer 2 represents 1.46% (401/27,462) of the total nodes and 7.99% (1,983/24,802) of the total edges in the _A. thaliana_ dataset. The average clustering coefficients for each organisms layer 2 were nearly identical with _A. thaliana_ displaying a 0.59 clustering coefficient and _S. cerevisiae_ displaying a 0.58 clustering coefficient. A total of 158 homologous edges were identified between the two layer 2's representing 4.72% (158/3,345) of the total _S. cerevisiae_ layer 1 edges, and 7.96% (158/1,983) of the total _A. thaliana_ layer 2 edges. 

```{r,echo=FALSE,fig.cap="Layer 2 combined index score statistics.",out.width="90%",out.height="60%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/L2_stats_updated_fig.png")
```

# Discussion {.unnumbered}

## Layer 0 Analysis {.unnumbered}
Due to the methodology of curating _A. thaliana_ layer 0 proteins from the list of _S. cerevisiae_ layer 0 proteins, a comparison between the two layers should be treated as a control case as opposed to experimental results. One of the most obvious discrepancies between the two layers is a marked difference between edge counts. While _S. cerevisiae_ layer 0 contains 166 edges, _A. thaliana_ layer 0 contains only 35. There are several possible explanations for this discrepancy. It is possible that edges in the _A. thaliana_ dataset were highly dependent on homology data which would have reduced the number of edges included when homology data was removed from the edge confidence calculation. Likewise, it is also possible that _A. thaliana_ edges had a lower confidence scores overall and, combined with a further reduction of confidence due to homology data removal, were greatly reduced in number by the 900 confidence score threshold. Another possible explanation is that the nodes in _A. thaliana_ layer 0 simply have less edges connecting them to other layer 0 nodes. This explanation is supported by the relatively low clustering coefficient displayed by _A. thaliana_ layer 0. The low clustering coefficient and relatively disconnected nodes in _A. thaliana_ layer 0 indicate that the individual connected components are not directly interacting with each other as they are in the _S. cerevisiae_ layer 0 and that additional regulatory elements are required to bridge the connected components. Conversely, the highly connected nature of the _S. cerevisiae_ layer 0 subgraph indicates that the literature review process was successful in identifying core proteins involved in the _S. cerevisiae_ homotypic vacuole fusion process. 


## Layer 1 Analysis {.unnumbered}
Layer 1 is the most experimentally relevant layer out of the three. While layer 0 is moreso representative of choices in methodology, layer 1 serves as somewhat of a middle ground where each node is likely involved in homotypic vacuole fusion yet was not curated directly through the literature review process. Comparing the top 20 proteins of each layer 1 subgraph by combined network importance score reveals a striking difference in protein function between the two organisms. While the majority of the high importance nodes in the _S. cerevisiae_ subgraph are subcomponents of the various protein complexes known to be essential for homotypic vacuole fusion, all of the top 20 high importance nodes in the _A. thaliana_ layer 0 subgraph are involved in lipid synthesis or modification. This incongruity is likely representative of an outsized role of regulatory lipids in the process of _A. thaliana_ homotypic vacuole fusion in comparison to _S. cerevisiae_ homotypic vacuole fusion. This conclusion is supported by the fact that there are fundamental differences between the phospholipid biosynthetic pathways of _A. thaliana_ and _S. cerevisiae_. Furthermore depletion of PI3P, an essential regulatory lipid in the _S. cerevisiae_ homotypic vacuole fusion process, has been shown to rapidly induce _A. thaliana_ guard cell vacuole fusion. 

## Layer 2 Analysis {.unnumbered}
Layer 2 was determined to not be experimentally relevant as the majority of proteins added in the conversion between layer 1 and layer 2 were not implicated in the vacuole fusion process. Furthermore, the majority of the top 20 high importance nodes in each layer 2 represent proteins which are not known to affect vacuole fusion. This conclusion is further supported by the low combined index scores of the top 20 proteins in each layer 2 subgraph. These scores show that nodes in the top 20 importance metric score relatively high in one metric but not others, indicating that the network lacks true 'hubs' and that the proteins in each subgraph are more likely to be representative of multiple distinct signaling pathways rather than the process of homotypic vacuole fusion. 

```{r, results='asis', echo=FALSE}
cat("\\newpage")
```

```{r,echo=FALSE,fig.cap="S. Cerevisiae layer statistics",out.width="100%",out.height="43%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/SC_hist_updated.png")
```

```{r,echo=FALSE,fig.cap="A. Thaliana layer statistics",out.width="100%",out.height="43%"}
knitr::include_graphics("/Users/adenobrien/Desktop/Thesis_Doc/figure/AT_hist_updated.png")
```

## Data Availability and Quality {.unnumbered}
As with any computation based study, the quality and quantity of data is extremely important. All data used in this study was downloaded from the STRING database which collects both known and predicted protein protein interactions (PPI's). STRING draws from a number of data sources when collecting PPI's. These sources include automated text mining of the scientific literature, computational interaction predictions from co-expression, conserved genomic context, databases of interaction experiments and known complexes/pathways from curated sources. String assigns each PPI a confidence score with higher scores signaling a higher confidence that the two proteins are interacting in vivo. As a single PPI may have multiple sources of evidence, the confidence score is calculated as an average of all evidence channels. However, to account for the probability that any two random proteins may interact, a prior value of 0.41 is integrated into each evidence channel score and must be removed and re-applied when calculating the overall confidence score of an interaction. As mentioned above, one of the evidence sources used by STRING to calculate confidence scores is conserved genomic context. Out of the 14 individual evidence channels used to calculate the confidence score, 6 consist of interaction data inferred from homologous PPI's observed in other species. While evidence obtained through homology is by no means inaccurate or invalid, these evidence channels were excluded from the data used in this analysis. As the goal of this experiment is to compare homologous PPI's present in both _A. thaliana_ and _S. cerevisiae_, including PPI's inferred through homology introduces potentially confounding data. For example, If a PPI in _S. cerevisiae_ was inferred through an observed interaction between homologous proteins in _D. melanogaster_ , the STRING data would include that interaction in the _S. cerevisiae_ PPI dataset. If those two proteins also interact in the _A. thaliana_, this interaction would be classified as homologous between _A. thaliana_ and _S. cerevisiae_ despite the data actually being representative of a homologous PPI between _A. thaliana_ and _D. melanogaster_. Fortunately, STRING provides the script used for calculating confidence scores from individual evidence channels and evidence inferred through homology was easily removed. In addition to removing potentially confounding evidence channels, a thresholding mechanism was employed during the data cleaning process. The threshold was set at a somewhat arbitrary value of 900, meaning that only PPI's with a total confidence score of 900 or greater would be included in the dataset. A threshold of 900 is relatively high in the context of STRING confidence scores and was chosen with the intention that all PPI's included in the dataset would have strong evidence backing their interaction in vivo. While removing homology derived evidence and thresholding by confidence scores reduce confounding data and increase data quality respectively, these two measures also reduce the total number of PPI's in the dataset. This potentially means removing PPI's that do occur in vivo, yet don't have sufficient supporting evidence in the literature. This is unfortunately unavoidable, yet must be taken into consideration when comparing the two PPI networks.

## Layer Based Analysis {.unnumbered}
The decision to perform independent analyses of each layer was a natural consequence of the data pruning process. After re-calculating confidence scores and thresholding the STRING data, additional processing was needed to isolate the subnetwork in each organisms dataset containing proteins likely to be implicated in the homotypic vacuole fusion process. While there are multiple methods for isolating subgraphs, a neighborhood-based approach seemed to be the best fit for the purpose of this experiment. By expanding the subgraph based on the neighbors of the nodes in the current subgraph, all edges between all nodes in the subgraph are preserved which increases the potential number of observed homologous edges. Additionally, by confining the subgraph to a maximum of two 'hops', the longest distance between any peripheral node and a layer 0 node is two edges, which increases the likelihood that peripheral nodes are in some way invloved in the homotypic vacuole fusion process. Initially, the process of determining the initial protein set for each organism was identical, perform a literature review and collate a list of proteins which have been experimentally verified to participate in the process of homotypic vacuole fusion. However, after completing _S. cerevisiae_ literature review, it became apparent that a similar process was not feasible for _A. thaliana_. _S. cerevisiae_ is considered to be the model organism for studying the process of homotypic vacuole fusion and a large body of work is devoted to the study of this particular process in both isolated _S. cerevisiae_ vacuoles and living _S. cerevisiae_ models. Conversely, there is little research devoted to the process of homotypic vacuole fusion in _A. thaliana_ and even less research focused on the precise mechanism of  _A. thaliana_ guard cell homotypic vacuole fusion. Therefore, in lieu of compiling a initial protein set for _A. thaliana_ through literature review, an initial set of _A. thaliana_ proteins was inferred by compiling all proteins in the _S. cerevisiae_ initial protein set and selecting those with homologous proteins in _A. thaliana_. This decision is justified by substantial evidence pointing towards the highly conserved nature of homotypic vacuole fusion across all eukaryotes. In theory, if the homotypic vacuole fusion process is highly conserved, then the core homologous proteins in _S. cerevisiae_ should, to an extent, represent proteins highly involved in the _A. thaliana_ homotypic vacuole fusion process. 

## Node Importance Metrics {.unnumbered}
While a host of node importance metrics are commonly used in network analysis, several considerations were taken into account when selecting the three importance metrics used in this experiment. The three selected importance metrics represent three general methods for assessing node importance, degree, clustering, and centrality. Degree centrality represents the number of neighbors connected to each node. Nodes with higher degree are considered to be more important in the context of the network as they represent a higher proportion of the PPI's than nodes with lower degree. To assess the clustering coefficient of each node a standard clustering coefficient algorithm was employed. The clustering coefficient algorithm counts the number of neighbors connected to any one node and uses that number to calculate the maximum number of edges that could exist between each neighbor. The maximum number of edges between neighbors is then divided by the observed number of edges between neighbors to compute the clustering coefficient. Clustering coefficient measures how interconnected a node is within its immediate neighborhood. This metric is useful for identifying biologically relevant subnetworks such as protein complexes as each protein within the complex interacts with most if not all of the other proteins in the complex, increasing the clustering coefficient for all proteins in the complex. Computing node centrality was slightly more challenging due to the _A. thaliana_ network being composed of multiple disconnected components. While degree and clustering calculations are not confined to fully connected networks, many centrality measures assume a fully connected network. Therefore, the harmonic centrality algorithm was used to compute node centrality. The harmonic centrality algorithm assigns a centrality value to a node equal to the sum of the reciprocals of each shortest path between a node and every other node in the graph. If there is no path between a node and another node in the graph, the reciprocal value of that path is set to 0 and does not affect the calculation. Centrality is used to identify nodes which are likely important to the flow of information throughout the network. 

## Protein Protein Interaction Networks {.unnumbered}
The PPI networks used in this experiment represent a subset of the data collected from the analysis of the signaling pathways governing homotypic vacuole fusion. Both networks are undirected graphs meaning that each PPI represents the interaction of two proteins, not the way in which one protein affects another. Additionally, the type of interaction is not included in the dataset, meaning that an interaction in which one protein phosphorylates another is indistinguishable from two proteins binding to form a complex. Furthermore, any non-protein regulatory element such as lipids are not represented in the PPI dataset. These factors limit the conclusions that can be drawn from PPI networks. However, for the purposes of this experiment, PPI networks provide sufficient data for the identification of broad potential similarities between the signaling networks involved in homotypic vacuole fusion between organisms. 

## Smith-Waterman Alignment {.unnumbered}
The Smith-Waterman alignment algorithm was employed to assess the likelihood that a protein from one organism was homologous to a protein from another organism. The algorithm compares segments of each proteins nucleotide sequence, taking into account gaps between aligned sequences, and calculates an overall alignment score which represents the similarity between each proteins nucleotide sequence. However, comparing SW scores between proteins is complicated as the SW score is dependant on the length of the proteins being compared. This means that a pair of relatively long proteins with a SW score of 200 are less similar than a pair of relatively short proteins with the same score. It may be possible to normalize the SW score of each protein pair based on length, however due to the time constraints of this project, this step was not performed and all proteins identified via SW alignment were designated as homologous regardless of SW score. 

